libraries/plugins/blockchain_history/account_history_api.cpp

```diff
@@ -23,8 +23,8 @@ class account_history_api_impl
     {
     }
 
-    template <typename history_object_type>
-    std::map<uint32_t, applied_operation> get_history(const std::string& account, uint64_t from, uint32_t limit) const
+    template <typename history_object_type, typename fill_result_functor>
+    void get_history(const std::string& account, uint64_t from, uint32_t limit, fill_result_functor& funct) const
```

Comment:
Add additional `get_history` overload for common case (which returns `std::map<uint32_t, applied_operation>`)